util/wait-for-services.sh

```diff
@@ -0,0 +1,394 @@
+#!/bin/bash
```

Comment:
copied and extended this script from the cypress workflows -> it now waits as part of run app dependencies until the db etc. are up. could also migrate cypress workflows etc. to use this script from util/